crates/providers/http-client/src/bin/httpclient.rs

```diff
@@ -1,156 +1,16 @@
-//! Httpclient capability provider
-//!
-//! This implementation is multi-threaded and requests from different actors
-//! use different connections and can run in parallel.
-use std::str::FromStr;
+use wasmcloud_provider_httpclient::HttpClientProvider;
 
-use reqwest::header as http;
-use reqwest::header::HeaderMap as HttpHeaderMap;
-use tracing::{error, instrument, trace, warn};
-use wasmbus_rpc::provider::prelude::*;
-use wasmcloud_interface_httpclient::{HttpClient, HttpClientReceiver, HttpRequest, HttpResponse};
-
-// main (via provider_main) initializes the threaded tokio executor,
-// listens to lattice rpcs, handles actor links,
-// and returns only when it receives a shutdown message
-//
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    provider_main(
-        HttpClientProvider::default(),
+    tracing_subscriber::fmt().init();
```

Comment:
Sure, will do! I do think that people would want to be able to configure the OTEL stuff on their providers (possibly writing  out to a different place), but we'll cross that bridge when we get there I guess!

- [ ] remove `tracing_subscriber::fmt().init()`

Reply:
Do you think people want to use different otel settings for providers vs hosts? The providers currently inherit their otel settings from the host settings

Reply:
Yeah I would imagine that some providers are more important than others and you'd want to know more about one in particular. 

Like I'm thinking of specifically someone running a provider that they wrote w/ the existing host and wanting to shuttle their o11y outside to some other service to see how this provider was doing specifically. I think of it like the host is mostly boring and the providers are where more action happens -- and they would change more frequently.

That said, it's totally reasonable to hook the external o11y solution to the host and filter out for the provider too... Definitely a "we can cross that bridge when we get there" thing.

Reply:
IMO single-provider-only traces aren't very useful, since you won't see anything preceding or following those spans. IME distributed tracing provides the most value when you have end-to-end traces

But if we're thinking of just logs, I do see what you mean. Do you want to file an issue related to this? I'll leave this thread for you to resolve either way